c/pthread-memsafety/list1_false-valid-deref.c

```diff
@@ -0,0 +1,49 @@
+#include <pthread.h>
+#include <stdlib.h>
+
+extern int __VERIFIER_nondet_int(void);
+
+struct item {
+  struct item *next;
+  struct item *data;
+};
+
+static void append(struct item **plist) {
+  struct item *item = malloc(sizeof *item);
```

Comment:
While this is valid C, it is highly confusing to have a variable with the same name(`item`) as the typedef. This might confuse people that try to understand why their verifier does not solve this task correctly. On the other hand it might also be worth having such a task in order to see whether the type and variable name handling of the verifier works the same way as demanded by the C standard.